client/js/_author/components/common/tiny_wrapper.jsx

```diff
@@ -89,7 +90,11 @@ export default class TinyWrapper extends React.Component {
           id={`${this.id}-tinymce`}
           content={this.props.text}
           config={this.tinyMCEConfig()}
-          onBlur={(e) => { this.props.onBlur(e.target.getContent(), e.target.isDirty()); }}
+          onBlur={(e) => {
+            const content = e.target.getContent();
+            const isChanged = e.target.isDirty() || _.isEmpty(content);
```

Comment:
Yeah it will update when you blur on an empty field. Which isn't ideal, how else can we guarantee we will update when all the text is removed though?